<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>642</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>965</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame{
	background-color: rgb(95, 95, 95);
	border-radius:30px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">#frame{
	background-color: rgb(95, 95, 95);
	border-radius:30px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>80</y>
         <width>241</width>
         <height>541</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_2{
	background-color: rgb(157, 157,157);
	border-bottom-left-radius:30px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="userButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>用户管理</string>
        </property>
       </widget>
       <widget class="QPushButton" name="faceButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>110</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>人脸数据</string>
        </property>
       </widget>
       <widget class="QPushButton" name="accessButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>门禁信息</string>
        </property>
       </widget>
       <widget class="QPushButton" name="setButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>211</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>242</x>
         <y>79</y>
         <width>701</width>
         <height>541</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>661</width>
          <height>501</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="userPage">
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;用户管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>581</width>
            <height>311</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="reButton">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>380</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
         <widget class="QPushButton" name="delButton">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>440</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addButton">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>440</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>提交数据</string>
          </property>
         </widget>
         <widget class="QPushButton" name="adButton">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>380</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>新增行</string>
          </property>
         </widget>
         <widget class="QPushButton" name="serchButton">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>440</y>
            <width>101</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>查询</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="serchLine">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>390</y>
            <width>271</width>
            <height>31</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>通过学号查询</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="facePage">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;人脸数据&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="huaLabel">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>110</y>
            <width>321</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="selectPho">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>340</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>选择图片</string>
          </property>
         </widget>
         <widget class="QPushButton" name="yesBotton">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>390</y>
            <width>93</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>录入数据</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="nameLine">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>290</y>
            <width>151</width>
            <height>25</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>录入人学号</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="idLine">
          <property name="geometry">
           <rect>
            <x>440</x>
            <y>240</y>
            <width>151</width>
            <height>25</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>录入人id</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="accessPage">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>10</y>
            <width>111</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;门禁信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QTableView" name="warnView">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>70</y>
            <width>531</width>
            <height>331</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="reButton_2">
          <property name="geometry">
           <rect>
            <x>292</x>
            <y>417</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="setPage">
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>10</y>
            <width>91</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;设置&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>420</y>
            <width>101</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>12</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>关于我的:</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_5">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>430</y>
            <width>71</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../sucai/icons/github.jpeg</normaloff>../../../sucai/icons/github.jpeg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>72</width>
            <height>40</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton_6">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>430</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../sucai/icons/bilibili.jpeg</normaloff>../../../sucai/icons/bilibili.jpeg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>72</width>
            <height>60</height>
           </size>
          </property>
         </widget>
         <widget class="QTableView" name="tableView_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>261</width>
            <height>331</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="delManage">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>320</y>
            <width>111</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>删除管理账号</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addManage">
          <property name="geometry">
           <rect>
            <x>490</x>
            <y>270</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>添加/修改账号</string>
          </property>
         </widget>
         <widget class="QPushButton" name="reManage">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>320</y>
            <width>111</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>刷新</string>
          </property>
         </widget>
         <widget class="QPushButton" name="addLine1">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>270</y>
            <width>111</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string>新增行</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="xiuPassword">
          <property name="geometry">
           <rect>
            <x>342</x>
            <y>200</y>
            <width>261</width>
            <height>41</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>密码:</string>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QFrame" name="frame_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>941</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">#frame_4{
	background-color: rgb(67,67,67);
	border-top-left-radius:30px;
	border-top-right-radius:30px;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>730</x>
          <y>0</y>
          <width>165</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../sucai/icons/minisize.png</normaloff>../../../sucai/icons/minisize.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../sucai/icons/restore.png</normaloff>../../../sucai/icons/restore.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../sucai/icons/maxsize.png</normaloff>../../../sucai/icons/maxsize.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../../../sucai/icons/close.png</normaloff>../../../sucai/icons/close.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>929</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>938</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>885</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>882</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>951</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>834</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>717</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
